cmd/interlock/run.go

```diff
@@ -163,18 +163,27 @@ func runAction(c *cli.Context) {
 		}
 	} else {
 		configPath := c.String("config")
+
 		if configPath != "" {
 			log.Debugf("loading config from: %s", configPath)
 
 			d, err := ioutil.ReadFile(configPath)
 			switch {
 			case os.IsNotExist(err):
+				log.Infof("Missing Interlock configuration: file=%s", configPath)
```

Comment:
Changed to `Error` level.

I tried a multiline string like this.

```
func missingConfig(configPath string)  {
	log.Errorf(`Missing Interlock configuration: file=%s
			    Use the run --config option to set a custom location for the configuration file
                Examples of an Interlock configuration file: url=https://github.com/ehazlett/interlock/tree/master/docs/examples`, configPath)
	log.Fatalf("config not found: file=%s", configPath)
}
```

But that produced the output.

```
# cmd/interlock/interlock -D run --config blah
INFO[0000] interlock 1.2.0-dev (HEAD)
ERRO[0000] Missing Interlock configuration: file=blah
	            Use the run --config option to set a custom location for the configuration file
	            Examples of an Interlock configuration file: url=https://github.com/ehazlett/interlock/tree/master/docs/examples
FATA[0000] config not found: file=blah
```

Not only does it have the funky indentation but I think the missing `ERRO` at the beginning of the lines could confuse log parsers setup to work with Interlock.